canonical/canonical.com
# Build /solutions/big-data-ai and modal

## Done

- Added /solutions/big-data-ai page as per [doc](https://docs.google.com/document/d/1UX4HyHpByhiG81_FZk71PxAGu6IE-GHfir3_18jNyYM/edit) and [design](https://www.figma.com/file/canUCAgMpSHPKCG183X81U/24.04-Canonical.com---Solutions?type=design&node-id=0%3A1&mode=design&t=rCaDqCZbKBsExtHr-1)
- Added corresponding modal as per [doc](https://docs.google.com/document/d/1ZDYZHdATF7Pqlmkq9s-ucN9YdCu7ZA648vlGkyQFApc/edit)

## QA

- View the site locally in your web browser at: https://canonical-com-1140.demos.haus/solutions/big-data-ai
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check page against doc and design 
- Open modal and check against doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7812 , https://warthogs.atlassian.net/browse/WD-6655
